src/ResourceManager/Websites/Commands.Websites/AzureRM.Websites.psd1

```diff
@@ -96,7 +96,9 @@ CmdletsToExport = 'Get-AzureRmAppServicePlan', 'Set-AzureRmAppServicePlan',
                'Reset-AzureRmWebAppPublishingProfile', 'Restart-AzureRmWebApp', 
                'Set-AzureRmWebApp', 'Start-AzureRmWebApp', 'Stop-AzureRmWebApp', 
                'Get-AzureRmWebAppSnapshot', 'Restore-AzureRmWebAppSnapshot',
-               'Get-AzureRmDeletedWebApp', 'Restore-AzureRmDeletedWebApp'
+               'Get-AzureRmDeletedWebApp', 'Restore-AzureRmDeletedWebApp',
+               'Enter-AzureRmWebAppContainerPSSession', 'Get-AzureRMWebAppContainerContinuousDeploymentUrl',
```

Comment:
The **M** in `Get-AzureRMWebAppContainerContinuousDeploymentUrl` should be lowercase. This might be causing the failure on Travis since **AzureRm** is replaced with **Az** when the cmdlets are tested in NetStandard.

Reply:
Thanks @MiYanni. Still failing.

I'm not familiar with Travis internals but I noticed that  "current directory" is set to "netcoreapp2.0" and "current executing assembly is "netstandard2.0".

 Write-Debug "current directory: /home/travis/build/Azure/azure-powershell/src/ResourceManager/Websites/Commands.Websites.Test/bin/Debug/netcoreapp2.0/"
 
 Write-Debug "current executing assembly: /home/travis/.nuget/packages/microsoft.azure.powershell.scenariotest.resourcemanager/1.1.2-preview/lib/netstandard2.0"

not sure if that means anything at all

Reply:
Also updated the mappings file. The test now passes successfully in Travis